shell/platform/windows/flutter_windows_view.cc

```diff
@@ -9,11 +9,10 @@
 namespace flutter {
 
 FlutterWindowsView::FlutterWindowsView(
-    std::unique_ptr<WindowBindingHandler> window_binding) {
-  resize_status_ = ResizeState::kDone;
-  resize_target_width_ = 0;
-  resize_target_height_ = 0;
-
+    std::unique_ptr<WindowBindingHandler> window_binding)
+    : resize_status_(ResizeState::kDone),
+      resize_target_width_(0),
+      resize_target_height_(0) {
```

Comment:
Initializer list is an improvement over the previous version, but I was referring to the C++11 feature of giving the initial value in the header where the value is declared: https://abseil.io/tips/61

Reply:
👍 done!